### PR TITLE
Get rid of unnecessary styles on service pages

### PR DESCRIPTION
**Pull request by domoscargin**
https://trello.com/c/lu1zDbsy/267-2-are-we-ready-to-migrate-buyer-frontend-to-govuk-frontend-v3

Trying to split out as much work as can be split out from the v3 migration branch (https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/1106).

We can replace all the classes in `_service_page.scss` with Design System classes, and this does that.

### CODE REVIEW

**Comment by lfdebrux** [comment]
### My review checklist

I've:

- [ ] looked at the code
- [ ] tested locally
- [ ] run functional tests locally

File: app/templates/service.html [hunk 1]
```diff
@@ -86,7 +86,7 @@ <h2 class="govuk-heading-m app-summary-list-heading" id="service-documents">Serv
       </li>
     {% endfor %}
     </ul>
-    <a class="govuk-link" href="#main-content" class="return-to-top">Return to top ↑</a>
+    <a class="govuk-link govuk-body" href="#main-content">Return to top ↑</a>
```
Comment (lfdebrux) [inline app/templates/service.html:89]: I wonder if we can get rid of this link altogether...

File: app/assets/scss/application.scss [hunk 2]
```diff
@@ -64,7 +64,6 @@ $govuk-global-styles: true;
 @import "_index-page.scss";
 @import "_search-page.scss";
 @import "_search-summary.scss";
-@import "_service-page.scss";
```
Comment (lfdebrux) [inline app/assets/scss/application.scss:67]: 🙌

File: app/templates/service.html (hunk 1 above)
Comment (domoscargin) [reply app/templates/service.html:89]: Considering we don't seem to use this anywhere else, and there's room for confusion with back to top links, I don't necessarily disagree...

**Review by lfdebrux — APPROVED** [review]
Great stuff 😁

**Pull request merged**